src/lib/Components/__stories__/DottedLine.story.tsx

```diff
@@ -0,0 +1,15 @@
+import { action, storiesOf } from "@storybook/react-native"
+import * as React from "react"
+import { View } from "react-native"
+
+import DottedLine from "../DottedLine"
+
+storiesOf("App Style/Dotted Line")
+  .addDecorator(story =>
+    <View accessibilityLabel="wrapperView" style={{ height: 10, marginTop: 60, marginLeft: 20, marginRight: 20 }}>
+      {story()}
+    </View>
+  )
+  .add("Flat White", () => {
```

Comment:
I think this title is the result from some copy-pasta, right? Can you name this something like ‘With default props’?

Reply:
Oh man. I kept looking at that name and thinking 'why did we name it this when it is neither flat nor white?'

Reply:
😂